docs/rules-engine/how-to-use/placeholders.md

```diff
@@ -135,15 +148,52 @@ You need to reference one or several CSS files from your application in the `cms
   ]
 }
 ```
+
 Those files will be loaded by the CMS to show the placeholder preview.
 Note that you could provide an empty file and update it with the dynamic content mechanism from AEM, to be able to reference the new classes afterwards.
 There is just no user-friendly editor available yet.
 You can include this file in your application using the style loader service in your app component:
+
 ```typescript
 this.styleLoader.asyncLoadStyleFromDynamicContent({id: 'placeholders-styling', href: 'assets/rules/placeholders.css'});
 ```
 
 ### How to create placeholders from AEM
+
 For this part, please refer to the Experience Fragments in DES documentation:
-https://dev.azure.com/AmadeusDigitalAirline/DES%20Platform/_wiki/wikis/DES%20Documentation/1964/Experience-Fragments-in-DES
+<https://dev.azure.com/AmadeusDigitalAirline/DES%20Platform/_wiki/wikis/DES%20Documentation/1964/Experience-Fragments-in-DES>
 
+## Manual usage of the Placeholder
+
+The Placeholder does not requires the Rules Engine to be used and can be integrated to your application independently.
+
+To do so you will need to import the `PlaceholderModule` in your application (as described in the [preview section](#/inside-an-application)) and describe the template to set to your application placeholders in the following manner:
+
+```typescript
+import { EffectsModule } from '@ngrx/effects';
+import { PlaceholderService, PlaceholderTemplateResponseEffect } from '@o3r/components';
+import { PlaceholderTemplateResponseEffect } from '@o3r/components/rules-engine';
```

Comment:
imported 2 times. You can remove it from the previous line

Reply:
Actually I need to remove this line :).
But thanks for the notice